src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        String randomColor = colorSupplier.getRandomColor().toString();
```

Comment:
Instead of calling toString() on the enum, you should use the name() method to get the String representation of the enum constants. This is to avoid issues in case the toString() method is overridden in the enum class.